src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
```

Comment:
Similarly, 'SHORT_TERM_DISCOUNT' could be renamed to 'DISCOUNT_FOR_3_DAYS_OR_MORE' to be more descriptive of the condition it applies to.